js/views/pageNavVw.js

```diff
@@ -33,7 +33,7 @@ module.exports = baseVw.extend({
     'click .js-aboutModal .js-tab': 'aboutModalTabClick',
     'click .js-navRefresh': 'navRefreshClick',
     'click .js-navAdminPanel': 'navAdminPanel',
-    'click .js-navProfileMenu a': 'closeNav',
```

Comment:
I'll undo this, it's a leftover from messing around with the profile menu bug, and unrelated to this PR. I think you fixed that bug in your notifications PR.